modules/installation-osp-accessing-api.adoc

```diff
@@ -6,25 +6,9 @@
 // * installing/installing_openstack/installing-openstack-user.adoc
 // * installing/installing_openstack/installing-openstack-user-kuryr.adoc
 
-ifeval::["{context}" == "installing-openstack-user"]
-:osp-user:
-endif::[]
-ifeval::["{context}" == "installing-openstack-user-kuryr"]
-:osp-user:
-endif::[]
-
 [id="installation-osp-accessing-api_{context}"]
 = Enabling access to the environment
```

Comment:
~This heading shouldn't start with a gerund, and I question whether it needs to be a separate module.~

You might want to add some notes that say that this stub is meant to be used with the other modules.